src/operation/run_command/mod.rs

```diff
@@ -1,17 +1,15 @@
 #[cfg(test)]
 mod test;
 
-use std::convert::TryInto;
-
 use bson::{RawBsonRef, RawDocumentBuf};
+use std::convert::TryInto;
```

Comment:
nit: can we revert this change? usually imports from std go at the top, then imports from external crates, then internal imports (with a blank line between each section)